README.md

```diff
@@ -269,6 +283,14 @@ The LIVE indicator only appears during a LIVE streaming video playback. This wil
 
 > [LIVE Indicator Color](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)
 
+#### Animations customization and disabling
+
+We've added some new animations for our player controls. So in case you want to change their duration, or to turn them off, you will have that opportuinity. These animations are added only to the default controls view controller. Check our playgrounds to find out how to turn off these animations and how to change the duration.
+
+##### _Tutorial Sample:_
+
+> [Animations customization](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)
```

Comment:
change:
+> [Animations customization](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)

to:
+> [Animations Customization](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)